native/android-jsi/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.nozbe.watermelondb.jsi">
+</manifest>
```

Comment:
```suggestion
<manifest package="com.nozbe.watermelondb.jsi" />
```

Reply:
fixed in https://github.com/Nozbe/WatermelonDB/pull/693/commits/2cbf4e017607f44d99e8e8c064ba1f01eab9bce1